src/components/Scrollable/Scrollable.tsx

```diff
@@ -130,6 +130,8 @@ export class Scrollable extends Component<ScrollableProps, State> {
             {...scrollable.props}
             {...rest}
             ref={this.setScrollArea}
+            // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
+            tabIndex={0}
```

Comment:
Did we want scrollable in the tab order or should this be -1? (focusable but not in the tab order)